lldb/tools/lldb-server/lldb-platform.cpp

```diff
@@ -455,35 +455,29 @@ int main_platform(int argc, char *argv[]) {
   lldb_private::Args inferior_arguments;
   inferior_arguments.SetArguments(argc, const_cast<const char **>(argv));
 
-  Socket::SocketProtocol protocol = Socket::ProtocolUnixDomain;
```

Comment:
There is a path when protocol remains undefined inside parse_listen_host_port().